package/MDAnalysis/analysis/align.py

```diff
@@ -512,12 +512,26 @@ def __init__(self, mobile, reference, select='all', filename=None,
             Last frame of trajectory to analyse, Default: -1
         step : int, optional
             Step between frames to analyse, Default: 1
+        in_memory : boolean, optional
+            *Permanently* switch `mobile` to an in-memory trajectory
+            so that alignment can be done in-place, which can improve
+            performance substantially in some cases. In this case, no file
+            is written out (`filename` and `prefix` are ignored) and only
+            the coordinates of `mobile` are changed in memory.
+        Default:
```

Comment:
You don't need to specify the default here. The original function signature with all default values will be shown in docs.